test/test.js

```diff
@@ -475,6 +475,18 @@ test('can override stream payload content-length header', (t) => {
   })
 })
 
+test('can override stream payload content-length header without request content-length', (t) => {
+  t.plan(1)
+  const dispatch = function (req, res) {
+    res.writeHead(200, { 'Content-Type': 'text/plain' })
+    res.end(req.headers['content-length'])
```

Comment:
You should do an assertion here rather than returning it.

Reply:
```js
const dispatch = function (req, res) {
    res.writeHead(200, { 'Content-Type': 'text/plain' })
    t.equal(req.headers['content-length'], '2')
}
```